server/routes/api.js

```diff
@@ -0,0 +1,80 @@
+const router = require('express').Router()
+const models = require('../models')
+
+const isAuthenticated = function (req, res, next) {
+	if (req.isAuthenticated()) return next()
+	res.redirect('/')
+}
+
+// Retrieves a summary list of notebooks in json format
+router.get('/notebook', (req, res) => {
+	models.Notebook.findAll().then(notebooks => {
+    if (notebooks && Object.keys(notebooks).length > 0)
+			res.json({ success: true, notebooks })
+		else
+			res.status(400).json({ success: false, error: "No notebooks found" })
```

Comment:
I don't think the lack of notebooks is necessarily an error here (even if it were, I would just return 404).